packages/framework/theming-react-native/src/BaselinePlatformDefaults.ts

```diff
@@ -28,6 +23,21 @@ function _defaultTypography(): ITypography {
       monospace: 'System',
       sansSerif: 'System',
       serif: 'System'
+    },
+    variants: {
+      captionStandard: { face: 'primary', size: 'caption', weight: 'regular' },
+      secondaryStandard: { face: 'primary', size: 'secondary', weight: 'regular' },
+      secondarySemibold: { face: 'primary', size: 'secondary', weight: 'semiBold' },
+      bodyStandard: { face: 'primary', size: 'body', weight: 'regular' },
+      bodySemibold: { face: 'primary', size: 'body', weight: 'semiBold' },
+      subheaderStandard: { face: 'primary', size: 'subheader', weight: 'regular' },
+      subheaderSemibold: { face: 'primary', size: 'subheader', weight: 'semiBold' },
+      headerStandard: { face: 'primary', size: 'header', weight: 'regular' },
+      headerSemibold: { face: 'primary', size: 'header', weight: 'semiBold' },
+      heroStandard: { face: 'primary', size: 'hero', weight: 'regular' },
+      heroSemibold: { face: 'primary', size: 'hero', weight: 'semiBold' },
+      heroLargeStandard: { face: 'primary', size: 'heroLarge', weight: 'regular' },
+      heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: 'semiBold' }
```

Comment:
Given the discussion earlier about control text size, I wonder if we will want to add some extra variant values for things like controlText.  This allows a common reference point and allows people to author it directly.  Just a thought.  (but not something to address for this review)